src/openrct2/rct2/T6Importer.cpp

```diff
@@ -169,7 +169,12 @@ class TD6Importer final : public ITrackImporter
                 _stream.SetPosition(_stream.GetPosition() - 1);
                 _stream.Read(&t6TrackElement, sizeof(rct_td46_track_element));
                 TrackDesignTrackElement trackElement{};
-                trackElement.type = t6TrackElement.type;
+
+                track_type_t trackType = t6TrackElement.type;
+                if (trackType == TrackElemType::RotationControlToggleAlias && !TrackTypeIsBooster(td->type, trackType))
```

Comment:
See also conversion at L1534 of TrackDesign.cpp. That conversion should move into this file.